docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker-in-Docker ("dind") in a kata container.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
+
+## Background
+
+Docker in Docker ("DinD") is the colloquial name for the ability to run `docker` from inside a container (whether provided by `docker` or otherwise) pattern for running `docker` from *inside* an existing `docker` container. You can read more about Docker-in-docker at the following links:
+
+- [The original announcement of DinD](https://www.docker.com/blog/docker-can-now-run-within-docker/)
+- [`docker` image DockerHub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
+
+**NOTE** While in the past kata containers did not contain the [`overlay` kernel module (aka OverlayFS)][overlayfs], it *now does* with the merging of the following pull requests:
+
+- [PR #1384](https://github.com/kata-containers/kata-containers/pull/1384)
+- [PR #716](https://github.com/kata-containers/packaging/pull/716)).
+
+`overlayfs` is the preferred containerization storage driver for linux systems ([Docker in particular](https://docs.docker.com/storage/storagedriver/select-storage-driver)), and as such, using it is often desired over other alternatives.
+
+[overlayfs]: https://www.kernel.org/doc/html/latest/filesystems/overlayfs.html
+
+## Why DinD containers require special measures
+
+Running containers DinD-capable containers with Kata takes more effort than normal due to the fact that `VOLUME`s specified in `Dockerfile`s run by kata containers are given the `kataShared` mount type by default, as it is what is used for the root directory `/`:
```

Comment:
Please always refer to "Kata Containers".

Reply:
- [ ] "kata" -> "Kata Containers" @ L31